appinventor/appengine/src/com/google/appinventor/client/editor/blocks/BlocklyPanel.java

```diff
@@ -702,6 +702,7 @@ private native void initWorkspace(String projectId, boolean readOnly, boolean rt
       }
     }.bind(workspace));
     this.@com.google.appinventor.client.editor.blocks.BlocklyPanel::workspace = workspace;
+    workspace.setVisible(false);  // The workspace is invisible by default
```

Comment:
I'm not sure if this is necessary but decided to leave it out of an abundance of caution.